plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/InviteCommand.java

```diff
@@ -68,4 +117,41 @@ public void execute(@CanManage User deputy, Guild guild, String[] args) {
         invitedUser.sendMessage(formatter.format(this.messages.inviteToInvited));
     }
 
+    public static final class InviteCommandCompleter implements Completer {
+
+        private final PluginConfiguration configuration;
+        private final UserManager userManager;
+
+        public InviteCommandCompleter(PluginConfiguration configuration, UserManager userManager) {
+            this.configuration = configuration;
+            this.userManager = userManager;
+        }
+
+
+        @Override
+        public String getName() {
+            return "invite-command";
+        }
+
+        @Override
+        public List<String> apply(Context context, String prefix, Integer limit) {
```

Comment:
Zinlineuj to jakoś żeby po prostu był return

Reply:
Mogę robić arraylist z "*" jako wartością i dodawać do niego pozostałe rzeczy, ale nie wiem na ile to jest możliwe. Próbowałem inlinować to ale miałem z tym problem

Reply:
Jak użyjesz `Stream.concat` z mojego review to problem magicznie się rozwiązuje.

Reply:
Fixed - Użyłem PandaStream#concat z review Kamila